From 3ff6185ecacb81b747b95eba8441396e40989d21 Mon Sep 17 00:00:00 2001
From: mati865 <mati865@gmail.com>
Date: Thu, 8 Sep 2016 14:18:56 +0200
Subject: [PATCH 11/13] fix building with --with-intl=system-icu

'Linux doesn't need icudata, guess we don't need it either'
---
 deps/v8/BUILD.gn         | 7 ++-----
 deps/v8/src/d8.gyp       | 5 -----
 deps/v8/tools/gyp/v8.gyp | 5 -----
 3 files changed, 2 insertions(+), 15 deletions(-)

diff --git a/deps/v8/BUILD.gn b/deps/v8/BUILD.gn
index 28aca24..e40c031 100644
--- a/deps/v8/BUILD.gn
+++ b/deps/v8/BUILD.gn
@@ -238,7 +238,7 @@ config("toolchain") {
   }
   if (v8_target_arch == "s390") {
     defines += [ "V8_TARGET_ARCH_S390" ]
-  } 
+  }
   if (v8_target_arch == "s390x") {
     defines += [
       "V8_TARGET_ARCH_S390",
@@ -251,7 +251,7 @@ config("toolchain") {
   if (v8_target_arch == "x64") {
     defines += [ "V8_TARGET_ARCH_X64" ]
   }
-     
+
   if (is_win) {
     defines += [ "WIN32" ]
     # TODO(jochen): Support v8_enable_prof.
@@ -1773,9 +1773,6 @@ source_set("v8_base") {
 
   if (v8_enable_i18n_support) {
     deps += [ "//third_party/icu" ]
-    if (is_win) {
-      deps += [ "//third_party/icu:icudata" ]
-    }
 
     # TODO(jochen): Add support for icu_use_data_file_flag
     defines += [ "ICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_FILE" ]
diff --git a/deps/v8/src/d8.gyp b/deps/v8/src/d8.gyp
index 7499d6d..9698212 100644
--- a/deps/v8/src/d8.gyp
+++ b/deps/v8/src/d8.gyp
@@ -96,11 +96,6 @@
             '<(icu_gyp_path):icuuc',
           ],
         }],
-        ['OS=="win" and v8_enable_i18n_support==1', {
-          'dependencies': [
-            '<(icu_gyp_path):icudata',
-          ],
-        }],
       ],
     },
     {
diff --git a/deps/v8/tools/gyp/v8.gyp b/deps/v8/tools/gyp/v8.gyp
index 45c7c8f..79de18f 100644
--- a/deps/v8/tools/gyp/v8.gyp
+++ b/deps/v8/tools/gyp/v8.gyp
@@ -1637,11 +1637,6 @@
             '../../src/i18n.h',
           ],
         }],
-        ['OS=="win" and v8_enable_i18n_support==1', {
-          'dependencies': [
-            '<(icu_gyp_path):icudata',
-          ],
-        }],
         ['icu_use_data_file_flag==1', {
           'defines': ['ICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_FILE'],
         }, { # else icu_use_data_file_flag !=1
-- 
2.9.1

